raiden-ts/src/transport/epics/rooms.ts

```diff
@@ -64,23 +60,24 @@ function inviteLoop$(
   { pollingInterval, httpTimeout }: RaidenConfig,
   { log }: Pick<RaidenEpicDeps, 'log'>,
 ) {
-  // defer here ensures invite is re-done on repeat (re-subscription)
-  return timer(pollingInterval).pipe(
-    mergeMap(() =>
-      defer(() => matrix.invite(roomId, userId)).pipe(
-        catchError((err) => (log.warn('Error inviting', err), EMPTY)),
-        // while shouldn't stop (by unsubscribe or takeUntil)
-        repeatWhen((completed$) => completed$.pipe(delay(httpTimeout))),
-      ),
-    ),
-    // takeIf will unsubscribe from repeat loop if user joins room, re-subscribe while/if not
-    takeIf(
-      fromEvent<[MatrixEvent, RoomMember]>(matrix, 'RoomMember.membership').pipe(
-        pluck(1),
-        filter((member) => member.roomId === roomId && member.userId === userId),
-        startWith(matrix.getRoom(roomId)?.getMember(userId)),
-        map((member) => member?.membership !== 'join'),
-        distinctUntilChanged(),
+  return timer(pollingInterval, httpTimeout).pipe(
+    exhaustMap(() =>
+      defer(async () => {
+        const membership = matrix.getRoom(roomId)?.getMember(userId)?.membership;
```

Comment:
Do we need rate limiting error handling here?

Reply:
No, the `catchError` and `timer` will loop through it and reinvite on next loop.